packages/web-components/fast-foundation/src/breadcrumb/breadcrumb.ts

```diff
@@ -25,11 +25,20 @@ export class Breadcrumb extends FASTElement {
                 this.slottedBreadcrumbItems.length - 1
             ];
 
+            this.resetItemSeparator();
             this.removeLastItemSeparator(lastNode);
```

Comment:
Could these two be combined to be a bit more efficient? Consider that you run through each item and set all separators to true and *then* you go find the last one and change it. Could these be combined into a single method which is more efficient?

Reply:
yeah, I will combine this

Reply:
updated